### DISCUSSION TITLE
Username in reset password email template

### DISCUSSION THREAD

**Discussion by dileepbc3141** [supabase/supabase#15829] [Questions] [answered]
is it possible to include username in the reset password email template .

I know supabase provides email template message variables :

```
{{ .ConfirmationURL }} : URL to confirm the password reset
{{ .Token }} : The 6-digit numeric email OTP
{{ .TokenHash }} : The hashed token used in the URL
{{ .SiteURL }} : The URL of the site
{{ .Email }} : The users email address
```
is it possible to configure this to include username.

**Comment by GaryAustin1** [comment, marked as answer]
You can try {{.Data.username}} if username is in user metadata.   That works for the magic link template at least.

**Reply by dileepbc3141** [reply]
thanks . your approach works
